src/dotnet-scaffolding/dotnet-scaffold/Aspire/ScaffoldSteps/AddAspireConnectionStringStep.cs

```diff
@@ -55,14 +55,24 @@ public override Task<bool> ExecuteAsync(ScaffolderContext context, CancellationT
         if (string.IsNullOrEmpty(appSettingsFile) || !_fileSystem.FileExists(appSettingsFile))
```

Comment:
Seems strange to check if the file exists, given we sourced the file from disk.

```suggestion
        if (string.IsNullOrEmpty(appSettingsFile))
```

Reply:
done!